docs(angular sample): Add css import step to the readme (#13925)
Esri/calcite-components: examples/components/angular/README.md

```sh
cp -r node_modules/@esri/calcite-components/dist/cdn/assets/ ./public
```

> ### Import the styles

Reviewer: nitpick: Should this be CSS styles?

```suggestion
### Import CSS styles
```

> Angular does not automatically load `.css` assets from external libraries. To ensure all component styles load correctly, import Calcite’s CSS in your Angular app’s global stylesheet (`src/styles.css`):

Reviewer 2: Some tweaks to be in line with Maps SDK docs.
```suggestion
Angular does not automatically load `.css` assets from external libraries. To ensure all component styles load correctly, explicitly import Calcite’s CSS in the project's root stylesheet (`src/styles.css`):
```

> Angular does not automatically load `.css` assets from external libraries. To ensure all component styles load correctly, explicitly import Calcite’s CSS in the project's root stylesheet (`src/styles.css`):

Reviewer: Is there a link we could add for folks to learn more about why Angular doesn't automatically load in the css assets?

Author: 😭 Nope, I looked at the Angular docs and it doesn't explicitly mention their stance on this (at least I couldn't find it). From the discussion [Max's issue](https://github.com/angular/angular-cli/issues/31017) it seems like they believe it's not best practice but didn't link to any resource they might have to back that stance. Also Maps SDK doesn't have any links like that so I'm assuming they didn't find any either.

Reviewer: 😿 

Thanks for researching - marking the above as resolved!
